cypher/cypher-docs/src/docs/dev/ql/administration/indexes-for-search-performance/index.asciidoc

```diff
@@ -288,7 +322,7 @@ Listing indexes can be done with `SHOW INDEXES`, which will produce a table with
 | `+`
 
 | `type`
-| The IndexType of this index (`BTREE`, `FULLTEXT`, or `LOOKUP`).
+| The IndexType of this index (`BTREE`, `FULLTEXT`, `TEXT`, or `LOOKUP`).
```

Comment:
In most other places we mention `LOOKUP` before `TEXT`, which also happen to alphabetical order, maybe you want that order here too.